tools/NuGet/template-nuget/DynamoVisualProgramming.DynamoCoreNodesWPF.nuspec

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+    <metadata>
+        <id>DynamoVisualProgramming.DynamoCoreNodesWPF</id>
+        <version>$Version$</version>
+        <authors>Autodesk</authors>
+        <owners>Autodesk</owners>
+        <license type="expression">Apache-2.0</license>
+        <projectUrl>https://github.com/DynamoDS/Dynamo</projectUrl>
+        <icon>content\images\logo.png</icon>
+        <requireLicenseAcceptance>false</requireLicenseAcceptance>
+        <description>
+            Reference UI node types for Dynamo. It contains the following files:
+            1) UnitsUI.dll
+            2) CoreNodeModelsWpf.dll
+        </description>
+        <copyright>Copyright Autodesk 2023</copyright>
+        <dependencies>
+            <group targetFramework=".NETFramework4.8">
+                <dependency id="DynamoVisualProgramming.DynamoServices" version="$Version$" />
+                <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="$Version$" />
+                <dependency id="DynamoVisualProgramming.DynamoCoreNodes" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.WpfUILibrary" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.Core" version="$Version$"/>
+            </group>
+            <group targetFramework="net6.0">
+                <dependency id="DynamoVisualProgramming.DynamoServices" version="$Version$" />
+                <dependency id="DynamoVisualProgramming.ZeroTouchLibrary" version="$Version$" />
+                <dependency id="DynamoVisualProgramming.DynamoCoreNodes" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.WpfUILibrary" version="$Version$"/>
+                <dependency id="DynamoVisualProgramming.Core" version="$Version$"/>
+            </group>
+        </dependencies>
+    </metadata>
+    <!--for now we assume that that the working directory root is dynamo/bin/AnyCPU/Release-->
+    <files>
+        <file src="nodes\UnitsUI.dll" target="lib\net48" />
+        <file src="nodes\UnitsUI.xml" target="lib\net48" />
+        <file src="nodes\CoreNodeModelsWpf.dll" target="lib\net48" />
+        <file src="nodes\CoreNodeModelsWpf.xml" target="lib\net48" />
+
+        <!--not yet migrated <file src="..\..\NET60_Windows\Release\nodes\UnitsUI.dll" target="lib\net6.0" />
+        <file src="..\..\NET60_Windows\Release\nodes\UnitsUI.xml" target="lib\net6.0" />
+         <file src="..\..\NET60_Windows\Release\nodes\CoreNodeModelsWpf.dll" target="lib\net6.0" />
+        <file src="..\..\NET60_Windows\Release\nodes\CoreNodeModelsWpf.xml" target="lib\net6.0" />-->
```

Comment:
wait until merged.

Reply:
Any reason for not including other WPF-based nodes in `Watch3DNodeModelsWPF`, `PythonNodeModelsWPF`, `GeometryUIWPF`, etc.?

Reply:
there just wasn't a use case for them, and actually after @twastvedt's latest comment on the wiki I think I can also remove the `UnitsUI` nuget as well - so I will need to think about where to put `CoreNodeModelsWpf` a bit more.

I don't think we should ship these UI binaries unless absolutely needed especially because we break them and don't semantically version them.

Reply:
So it looks like as per Trygve's comment we need to split UnitsUI and add its nodemodels half into the DynamoCoreNodes package.